functions/exercises.py

```diff
@@ -57,10 +55,14 @@ def get_prod_of_list(num_list, absolute=False):
     :param absolute: Use absulute values
     :return:
     """
-    # INSERT YOUR CODE HERE
-    #
-    # Run this command to test your implementation:
-    #
+    product =  1
+    if absolute :
+        for num in num_list:
+            product *= abs(num)
+    else:
+        for num in num_list:
+            product *= num
+    return product

```

Comment:
```suggestion
    for num in num_list:
        product *= num
    if absolute :
            product = abs(product)
    return product
```

reducing code duplication is a great way to keep your code simple